src/js/search.js

```diff
@@ -51,19 +41,24 @@ class Search {
 		};
 
 		if (toggle) {
-			const clickHandler = function() {
+			const searchToggleClickHandler = function() {
 				if (transition) {
 					form.addEventListener(transition, transitionHandler);
 				} else {
 					setTimeout(transitionHandler, 300);
 				}
 
-				typeahead.hide();
+				headerEl.dispatchEvent(new CustomEvent('oHeader.searchToggle', {
+					bubbles: true,
+					detail: {
+						isOpen: getComputedStyle(form, null).getPropertyValue('visibility') === 'visible'
```

Comment:
I'm not saying remove the transition (though I think we should). I just don't want to spend any more time than the absolute minimum working around it